samples/cellular/nrf_cloud_coap_cell_location/README.rst

```diff
@@ -0,0 +1,124 @@
+.. _nrf_cloud_coap_cell_location:
+
+Cellular: nRF Cloud CoAP cellular location
+##########################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+This sample demonstrates how to use the `nRF Cloud CoAP API`_ for nRF Cloud's cellular location service on your device.
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-sample-yaml::
+
+.. include:: /includes/tfm.txt
+
+The sample requires an `nRF Cloud`_ account.
+Your device must be onboarded to nRF Cloud.
+If it is not, follow the instructions in `Device on-boarding <nrf_cloud_coap_cell_location_onboarding>`_.
+
+.. note::
+   This sample requires modem firmware v1.3.x or later for an nRF9160 DK, or modem firmware v2.x.x for the nRF9161 and nRF9151 DKs.
+
+Overview
+********
+
+After the sample initializes and connects to the network, it enters the single-cell mode and sends a single-cell location request to nRF Cloud using network data obtained from the :ref:`modem_info_readme` library.
+To enable the multi-cell mode, press **Button 1**.
+In the multi-cell mode, the sample requests for neighbor cell measurement using the :ref:`lte_lc_readme` library.
+If neighbor cell data is measured, the sample sends a multi-cell location request to nRF Cloud.
+Otherwise, the request is single-cell.
+In either mode, the sample sends a new location request if a change in cell ID is detected.
```

Comment:
```suggestion
The sample can work in two modes: single-cell mode and multi-cell mode.

After the sample initializes and connects to the network, it enters the single-cell mode and sends a single-cell location request to nRF Cloud.
For this purpose, the sample uses network data obtained from the :ref:`modem_info_readme` library.

In the multi-cell mode, the sample requests for neighbor cell measurement using the :ref:`lte_lc_readme` library.
If neighbor cell data is measured, the sample sends a multi-cell location request to nRF Cloud.
Otherwise, the request is single-cell.

In either mode, the sample sends a new location request if a change in cell ID is detected.
```